include/odp/api/spec/crypto.h

```diff
@@ -327,6 +327,9 @@ typedef struct odp_crypto_session_param_t {
 	 */
 	odp_crypto_key_t auth_key;
 
+	/** Authentication Initialization Vector (IV) */
+	odp_crypto_iv_t auth_iv;
+
```

Comment:
/** Cipher Initialization Vector (IV) */
odp_crypto_iv_t iv;

Shame that original cipher IV was not named "cipher_iv", but just "iv". I'm wondering if we should create an unnamed union of iv and cipher_iv, move all our code to use "cipher_iv" and then later on deprecate "iv" (some time after TigerMoth).